spec/models/sms/adapters/twilio_sms_adapter_spec.rb

```diff
@@ -54,6 +54,43 @@
     expect(msg.to).to be_nil
   end
 
+  context "broadcast message" do
+    let(:msg) { create(:sms_broadcast) }
+    let(:messages) { double(:twilio_messages, create: true) }
+    let(:client) { double(:twilio_client, messages: messages) }
```

Comment:
This is a nice way of preventing sending. Can you please change all the specs in this file only to use this method instead of relying on the `Rails.env.test?` check?